src/dns_private.tf

```diff
@@ -22,3 +22,22 @@ resource "azurerm_private_dns_a_record" "private_dns_a_record_db_nodo" {
   ttl                 = 60
   records             = var.dns_a_reconds_dbnodo_ips
 }
+
+resource "azurerm_private_dns_zone" "redis_pagopa_proxy_dns_zone" {
+  count               = var.pagopa_proxy_redis_private_endpoint_enabled ? 1 : 0
```

Comment:
`var.pagopa_proxy_redis_private_endpoint_enabled  && pagopa_proxy_enabled ? 1 : 0`

Reply:
it's correct to use only `pagopa_proxy_redis_private_endpoint_enabled`

for dns u can use a global var redis_private_endpoint_enabled not related to pagopa_proxy